### PR TITLE
GEOMESA-2238 - Minor dependency version updates

### PR DESCRIPTION
**Pull request by jnh5y**
Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -116,7 +116,7 @@
 
         <!-- logging properties - we expect logging jars to be provided -->
         <!-- 1.7.x is for hadoop 2 and accumulo 1.5.x -->
-        <slf4j.version>1.7.5</slf4j.version>
+        <slf4j.version>1.7.21</slf4j.version>
```
Comment (jnh5y) [inline pom.xml]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15184 (1.7.21)

File: pom.xml [hunk 2]
```diff
@@ -1445,7 +1445,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-pool2</artifactId>
-                <version>2.2</version>
+                <version>2.4.2</version>
```
Comment (jnh5y) [inline pom.xml]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14560 (2.4.2)

File: pom.xml [hunk 3]
```diff
@@ -1927,7 +1927,7 @@
             <dependency>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
-                <version>2.3.1</version>
+                <version>2.8.1</version>
```
Comment (jnh5y) [inline pom.xml]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14005 (2.8.1)

File: pom.xml [hunk 4]
```diff
@@ -2512,7 +2512,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.3.2</version>
+                <version>3.6</version>
```
Comment (jnh5y) [inline pom.xml]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14197 (3.6)

**Comment by jahhulbert-ccri** [comment]
where are these coming from?

File: pom.xml (hunk 1 above)
Comment (elahrvivaz) [reply pom.xml]: we'll need to open piggybacks, right?

File: pom.xml (hunk 2 above)
Comment (jnh5y) [reply pom.xml]: GeoServer 2.12.0 has version commons-pool-1.5.4.  (I.e., not comons pool2).

File: pom.xml (hunk 3 above)
Comment (jnh5y) [reply pom.xml]: GeoServer 2.12.0 has no gson jar.

File: pom.xml (hunk 4 above)
Comment (jahhulbert-ccri) [reply pom.xml]: this was baselined on hadoop before FYI

Comment (jnh5y) [reply pom.xml]: GeoServer has commons-lang-2.6.  (I.e., it should not overlap)

File: pom.xml (hunk 1 above)
Comment (jahhulbert-ccri) [reply pom.xml]: i think this is OK. it is likely baselined on accumulo 1.8.1

File: pom.xml (hunk 3 above)
Comment (jahhulbert-ccri) [reply pom.xml]: accumulo 1.8 is baselined on 2.7

File: pom.xml (hunk 1 above)
Comment (jnh5y) [reply pom.xml]: Approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16001

File: pom.xml (hunk 3 above)
Comment (jnh5y) [reply pom.xml]: Approved: 
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16003

File: pom.xml (hunk 2 above)
Comment (elahrvivaz) [reply pom.xml]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16002

File: pom.xml (hunk 4 above)
Comment (elahrvivaz) [reply pom.xml]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16004

File: build/cqs.tsv [hunk 5]
```diff
@@ -227,8 +227,8 @@ org.scalatra:scalatra-auth_2.11	2.3.0	compile
 org.scalatra:scalatra-common_2.11	2.3.0	compile
 org.scalatra:scalatra-json_2.11	2.3.0	compile
 org.scalatra:scalatra-swagger_2.11	2.3.0	compile
-org.slf4j:slf4j-api	1.7.5	compile
-org.slf4j:slf4j-log4j12	1.7.5	compile
+org.slf4j:slf4j-api	1.7.21	compile
+org.slf4j:slf4j-log4j12	1.7.21	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: @jnh5y we probably need a cq for this too - i'll open one

Comment (elahrvivaz) [reply build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16025

**Pull request merged**